pytorch_lightning/strategies/hpu.py

```diff
@@ -0,0 +1,73 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional
+
+import torch
+
+import pytorch_lightning as pl
+from pytorch_lightning.plugins.io.hpu_io_plugin import HPUCheckpointIO
+from pytorch_lightning.plugins.precision import PrecisionPlugin
+from pytorch_lightning.strategies.single_device import SingleDeviceStrategy
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+
+
+class HPUStrategy(SingleDeviceStrategy):
+    """Strategy for training on HPU devices."""
+
+    strategy_name = "hpu_single"
+
+    def __init__(
+        self,
+        device: int,
+        accelerator: Optional["pl.accelerators.accelerator.Accelerator"] = None,
+        checkpoint_io: Optional[HPUCheckpointIO] = None,
+        precision_plugin: Optional[PrecisionPlugin] = None,
+        hmp_params: Optional[str] = None,
+    ):
+
+        device = device
+        checkpoint_io = checkpoint_io or HPUCheckpointIO()
+        super().__init__(
+            accelerator=accelerator, device=device, checkpoint_io=checkpoint_io, precision_plugin=precision_plugin
+        )
+
+    @property
+    def is_distributed(self) -> bool:
+        return False
+
```

Comment:
isnt this already defined on `SingleDeviceStrategy` ? why re-define it here?

Reply:
followed TPU reference to explicitly state it so that strategy is specific in single card runs